docs/FAQ.md

```diff
@@ -0,0 +1,27 @@
+# FAQ
+
+Dit zijn de algemene veelgestelde vragen over het gebruik en de configuratie van het OpenWoo platform. Mocht u een vraag hebben die hier niet beantwoord wordt, of wilt u een vraag toegevoegd zien aan deze lijst, dan nodigen we u uit om een issue aan te maken op onze [GitHub-pagina](https://github.com/ConductionNL/woo-website-template/issues).
+
+**Q. Heb ik WordPress nodig om de OpenWoo.app te gebruiken?**
+
+A. Nee, het enige wat je nodig hebt is een bron. Dit zal veelal je zaaksysteem zijn, maar zou zelfs een Excel-lijst kunnen zijn.
```

Comment:
Nee, het enige wat je nodig `hebt, is` een bron. Dit zal veelal je zaaksysteem zijn, maar zou zelfs een Excel-lijst kunnen zijn.